interpreter/script/js.ml

```diff
@@ -10,20 +10,24 @@ let harness =
   "'use strict';\n" ^
   "\n" ^
   "let spectest = {\n" ^
-  "  print: console.log.bind(console)," ^
-  "  print_i32: console.log.bind(console)," ^
-  "  print_i32_f32: console.log.bind(console)," ^
-  "  print_f64_f64: console.log.bind(console)," ^
-  "  print_f32: console.log.bind(console)," ^
-  "  print_f64: console.log.bind(console)," ^
-  "  global_i32: 666," ^
-  "  global_f32: 666," ^
-  "  global_f64: 666," ^
-  "  table: new WebAssembly.Table({initial: 10, maximum: 20, element: 'anyfunc'})," ^
-  "  memory: new WebAssembly.Memory({initial: 1, maximum: 2})" ^
+  "  print: console.log.bind(console),\n" ^
+  "  print_i32: console.log.bind(console),\n" ^
+  "  print_i32_f32: console.log.bind(console),\n" ^
+  "  print_f64_f64: console.log.bind(console),\n" ^
+  "  print_f32: console.log.bind(console),\n" ^
+  "  print_f64: console.log.bind(console),\n" ^
+  "  global_i32: 666,\n" ^
+  "  global_f32: 666,\n" ^
+  "  global_f64: 666,\n" ^
+  "  table: new WebAssembly.Table({initial: 10, maximum: 20, element: 'anyfunc'}),\n" ^
+  "  memory: new WebAssembly.Memory({initial: 1, maximum: 2})\n" ^
   "};\n" ^
-  "\n" ^
-  "let registry = {spectest};\n" ^
+  "let spectest_handler = {\n" ^
```

Comment:
Nit: just name it handler, it wraps the registry, not the spectest module in it.

Reply:
Done.